packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeActionButton.tsx

```diff
@@ -32,10 +40,19 @@ export type FilterActionProps = {
 } & Partial<Pick<ReturnType<typeof useTree>, "getHierarchyLevelDetails">>;
 
 /** @internal */
-export function TreeActionButton({ show, label, action, icon }: TreeItemAction) {
+export function TreeActionButton({ show, label, action, icon, activeDescription }: TreeItemAction) {
   const { localizedStrings } = useLocalizationContext();
   const localizedLabel: string | undefined = localizedStrings[label as keyof typeof localizedStrings];
-  return <Tree.ItemAction visible={show} onClick={action} label={localizedLabel ?? label} icon={icon ?? placeholderIcon} />;
+  const localizedDescription: string | undefined = localizedStrings[activeDescription as keyof typeof localizedStrings];
+  return (
+    <Tree.ItemAction
+      render={<IconButton dot={localizedDescription ?? activeDescription} icon={icon ?? placeholderIcon} variant={"ghost"} label={localizedLabel ?? label} />}
```

Comment:
Do we still need `placeholderIcon`? I think we should make `icon` required and stop using the placeholder.

Reply:
I guess we don't need it, made icon required.